be/src/storage/tablet.cpp

```diff
@@ -1534,6 +1534,13 @@ bool Tablet::_contains_rowset(const RowsetId rowset_id) {
 }
 
 void Tablet::build_tablet_report_info(TTabletInfo* tablet_info) {
+    // primary key specified info will protected by independent lock context
+    // in TabletUpdates which means that those lock context can be sperated from
+    // the tablet meta_lock when trying to get the extra info
+    if (_updates) {
+        _updates->get_tablet_info_extra(tablet_info);
```

Comment:
if this is the case, shall we move this to the end and execute without meta lock? the seq will be the same  as the original one, just without the meta lock.